pkg/knowledge_base/edge_kb.go

```diff
@@ -58,6 +58,9 @@ type (
 		Destination core.Resource
 		// Routes refers to any api routes being satisfied by the edge
 		Routes []core.Route
+		// SourceRef denotes the source annotation being used during expansion or configuration
+		// This is a temporary field due to helm chart being the lowest level of kubernetes resource at the moment
```

Comment:
because multiple exec units can exist within a helm chart so the .Source.Refs doesnt really give us the information as to what we are connecting when we get to the dependency here. Realistically we would want the dependency to be to the manifest within the helm chart so we know what exec unit its tied to